runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -7778,9 +7776,9 @@ CISCTransform2PtrArraySet(TR_CISCTransformer *trans)
    return true;
    }
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
+
+
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there's no need to change these lines. May I ask you to change them back to the following?

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```